### PR TITLE
Update

### PR DESCRIPTION
**Pull request by gcedo**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -37,48 +36,60 @@ export default class SectionsCard extends React.Component {
...
+              {...commonProps}
+              target="_blank"
+              unstyled
+            >
+              {link.title}
+            </Button>
+          </li>
+        );
+      }
+      return (
```
Comment (keithamus) [inline src/index.js]: This isn't very DRY. Seems like you could make a props object and just assign target & append className and do one JSX render pass.

File: src/index.js [hunk 2]
```diff
@@ -1,103 +1,42 @@
...
+if (process.env.NODE_ENV !== 'production') {
+  const LinkType = React.PropTypes.shape({
+    href: React.PropTypes.string.isRequired,
+    title: React.PropTypes.string.isRequired,
+  });
+  SectionsCard.propTypes = {
+    data: React.PropTypes.shape({ // eslint-disable-line id-blacklist
+      blogs: React.PropTypes.arrayOf(LinkType),
+      media: React.PropTypes.arrayOf(LinkType),
+      sections: React.PropTypes.arrayOf(LinkType),
```
Comment (keithamus) [inline src/index.js]: Each of these could be `SectionCardList.propTypes.links` if you wanted 😄

File: src/parts/section-card-links.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+      unstyled: true,
+      ...link,
+    };
+    let buttonClassName = 'sections-card__link';
+    if (link.internal === false) {
+      commonProps.target = '_blank';
+      buttonClassName = `${ buttonClassName } sections-card__link--external`;
+    }
+    return (
+      <li className="list__item" key={`sections-card__link_${ i }`}>
```
Comment (keithamus) [inline src/parts/section-card-links.js]: Each `li` could also be its own component.

File: src/index.js (hunk 2 above)
Comment (gcedo) [inline src/index.js]: Ah super cool!

File: src/parts/section-card-links.js (hunk 3 above)
Comment (gcedo) [inline src/parts/section-card-links.js]: I think that would be over-splitting at the moment, it is a 45 lines component.

Comment (keithamus) [inline src/parts/section-card-links.js]: Its less about the LOC and more about making sure that if a function returns JSX, its a component. This gives us a super consistent API for things that return JSX.

**Pull request merged**